R/html_dependency.R

```diff
@@ -305,6 +305,15 @@ copyDependencyToDir <- function(dependency, outputDir, mustWork = TRUE) {
   files <- if (dependency$all_files) list.files(dir) else {
     unlist(dependency[c('script', 'stylesheet', 'attachment')])
   }
+  if (any(!file.exists(files))) {
+    stop(
+      sprintf(
+        "Can't copy dependency files that don't exist: '%s'",
+        paste(files, collapse = "', '")
+      ),
+      call. = FALSE
```

Comment:
Is there a particular reason to use `call. = FALSE` here? I know that some people like to use that in general for error messages, but unless there's a specific reason here, I'd rather leave it off to be consistent with the rest of the code here.

Reply:
> Is there a particular reason to use call. = FALSE here?

I think it reads slightly better, but I have no strong feelings, so I removed it